src/storage/src/hummock/mod.rs

```diff
@@ -208,7 +208,7 @@ impl HummockStorage {
 
     async fn get_compaction_group_id(&self, table_id: TableId) -> HummockResult<CompactionGroupId> {
         match tokio::time::timeout(
-            Duration::from_secs(10),
+            Duration::from_secs(200),
```

Comment:
?

Reply:
It should do retry. 10s is too short and will easily timeout. I'll leave it to 10 and let others fix this...

Reply:
I've opened an issue before. #4427